README.md

```diff
@@ -212,6 +211,51 @@ _Notes:_
 
 Compilation to iOS is supported on macOS targeting the iOS simulator (`--target x86_64-apple-ios`) and 64 bit ARM devices (`--target aarch64-apple-ios`). The ARM64**e** architecture is [not supported yet](https://github.com/rust-lang/rust/issues/73628).
 
+### For WebAssembly
+
+Install `emscripten` version 3.1.1 or superior.
+
+Build with the `wasm32-unknown-emscripten` target (`wasm32-unknown-unknown` is
+unsupported because it is [fundamentally incompatible with linking C code](https://github.com/rustwasm/team/issues/291#issuecomment-645482430):
+
+```bash
+export EMSDK=~/.asdf/installs/emsdk/3.1.1
+export EMCC_CFLAGS="-s ERROR_ON_UNDEFINED_SYMBOLS=0"
+
+cargo build --target wasm32-unknown-emscripten
+```
+
+The `EMSDK` environment variable must be set to the root of your `emscripten` SDK.
+
+In `EMCC_CFLAGS`, `-s ERROR_ON_UNDEFINED_SYMBOLS` is a
+[workaround](https://github.com/rust-lang/rust/issues/85821#issuecomment-969369677) to build with
+`emscripten > 2.0.9`.
+
+If you want to enable WebGL, you will also have to set `MAX_WEBGL_VERSION=2`:
+
+```bash
+export EMSDK=~/.asdf/installs/emsdk/3.1.1
+export EMCC_CFLAGS="-s ERROR_ON_UNDEFINED_SYMBOLS=0 -s MAX_WEBGL_VERSION=2"
+
+cargo build --target wasm32-unknown-emscripten --features gl
+```
+
+On MacOS there is a problem with the OS version of `ar` so you will have to install the GNU version from homebrew:
+
+```bash
+brew install binutils
+```
+
+Then prepend `binutils` path for the build:
+
+```bash
+export EMSDK=~/.asdf/installs/emsdk/3.1.1
+export EMCC_CFLAGS="-s ERROR_ON_UNDEFINED_SYMBOLS=0"
+export PATH="/opt/homebrew/opt/binutils/bin:$PATH"
```

Comment:
`brew install binutils` caused the executables to be installed in `/usr/local/opt/binutils/bin` on my macOS Big Sur (Intel) machine. Not sure if this has changed over time or may depend on other factors. `brew info binutils` saved me here.

Reply:
Ah yes it seems to depend on the CPU type:

> This script installs Homebrew to its preferred prefix (/usr/local for macOS Intel, /opt/homebrew for Apple Silicon and /home/linuxbrew/.linuxbrew for Linux)

https://docs.brew.sh/Installation

I added a note about it in a6c12d6d